### PR TITLE
Default: New jungletree_top texture

### PR DESCRIPTION
**Pull request by paramat**
screenshot_20160304_005050

new_jungletree_top_octagonal5_128

^ Submitted (branch not yet updated)

rosewood-stump-and-wastage-560x260

^ Rosewood

default_jungletree_top_128

^ Current texture

The colour palette of the lianas and bark is taken from the jungletree side texture.

For discussion of the structure of Minetest's jungletree node see https://github.com/minetest/minetest_game/issues/611#issuecomment-129177198 and c55's comment https://github.com/minetest/minetest_game/issues/611#issuecomment-129481251

index

^ Strangler fig

strangled

By blurring the side texture you can see the structure is 4 thick lianas (like the strangler fig in the photo) running along the node corners, this is confirmed by c55. Our previous 2 top textures have flattened the lianas down to a single pixel, along with the wrong bark colour. My texture gives the lianas some thickness by making them triangular in cross section. To make the structure fill the cube the trunk is therefore an octagon.

The junglewood planks texture is closest in appearence to Wenge and Rosewood. The tree shape seems to be closer to Rosewood than Wenge. So this texture is designed to look similar to those two trees.

0304 greenpeace 2013-03-04-at-2 05 31-pm 360

^ Wenge

I have re-arranged my own texture credits into categories.
I have removed the 'derived from ...' comments because they are making the file look messy, are not used by others, and i think not needed because many textures are edited versions of previous textures and this can be assumed, the credit just shows who edited a texture last.

### CODE REVIEW

**Comment by kilbith** [comment]
Its planks have a total different color. Big inconsistency.

Fortunally we're saved by texture packs...

**Comment by paramat** [comment]
I might try adjusting the colour of the centre to be closer to the wood texture.

**Comment by C1ffisme** [comment]
The Real Life texture does look a bit browner.

There seems to be quite some confusion on what tree the jungle tree is. It's probably worth solving at some point.

**Comment by Fixer-007** [comment]
Agree with kilbith

**Comment by paramat** [comment]
Updated to be similar to the junglewood texture, thanks kilbith.

**Comment by RHRhino** [comment]
@paramat I like the new texture, but could you smooth the bark of it a bit more. It currently looks a bit too flat.

**Comment by C1ffisme** [comment]
Looks really, really good. :+1:

**Comment by kilbith** [comment]
PixelBOX's textures, for comparison : 

screenshot_20160229_140304

**Comment by RHRhino** [comment]
@kilbith In my opinion the jungle leaves are better than the current ones, but the tree logs (green and light green) don't fit with the (dark brown) jungle wood texture.

**Comment by MoNTE48** [comment]
Pixelbox is better

**Comment by paramat** [comment]
Updated on request of Nore and ShadowNinja. The previous black pixels at the texture edges are now dark green to blend with the bark texture in the side texture, this makes the edges more continuous.

**Comment by VanessaE** [comment]
Frankly I like the existing texture much better than the proposed replacement.  It's too "hard edged".  No antialiasing.

**Comment by paramat** [comment]
I have a new idea for this, will update soon.

**Comment by paramat** [comment]
Updated with new texture.

**Comment by paramat** [comment]
Vanessa it's pixel art so doesn't necessarily need anti-aliasing. AA makes an image look like a low-res representation of a hi-res image, another approach is to simply consider it as pixel art that is itself and not a representation. I think the diagonal detials look okay, AA might make those details larger and blurry, currently it's clear and defined.

**Comment by VanessaE** [comment]
Real pixel art has, for most of its history (as in from early in the 8-bit era of computing), been about artistic style and expression, not usually about representing something realistically, and has almost always included at least basic use of anti-aliasing (in most cases, a hell of a lot of it).  Usually that's due to having a limited color palette, but in our case it's simply due to limited resolution.

That said, this is a tree, not a random, fictional machine.  It needs to actually look like a tree. :-)

Your top/center texture does not match well, style-wise, with the sides of the tree or the periphery of the top side, and, as the jungle tree is based on a strangler fig wrapped around something, the top of the tree needs to show the shape distortions the fig would cause to the center part.  The shape is too geometric.  To my eyes, it looks like a slightly-wrong octagon, rather than the slightly-distorted square shape the current texture has, and this looks wrong when used on a square trunk.

tl;dr:  The top part needs to be lower-contrast, maybe a little darker, distorted, and anti-aliased a little on the sharpest edges.

**Comment by C1ffisme** [comment]
@paramat I'm not sure I like that circle, though maybe if the white was a tad darker, it would fit in a bit more.

By the way, how do you take a screenshot from exactly the same angle every time?

**Comment by paramat** [comment]
> the top of the tree needs to show the shape distortions the fig would cause to the center part.

It does, the trunk is deformed to a trucated square.

> The shape is too geometric.

Our trees have perfect square trunks :)
I might attempt some anti-aliasing on this.

**Comment by nerzhul** [comment]
:-1:

**Comment by paramat** [comment]
My attitude has shifted, i will add AA to this, it has been requested by other devs.

**Comment by paramat** [comment]
Added screenshot and sample of AA version, not yet updated in branch.

**Comment by Ekdohibs** [comment]
:+1:

**Comment by PilzAdam** [comment]
:-1:, this texture has way too much contrast.

**Comment by ObaniGemini** [comment]
It's round in a real tree because the tree is round. Here it shall be a square, adding some circle in the texture just makes it awful to look. Also, the clearest part is too light to me.

:-1: 

Pixel Box's jungle trees are better because they are unified and good looking. Also, if you replace that greenish color with some dark-brown color, I think it would still fit well.

**Comment by paramat** [comment]
You misunderstand, read the first comment for why it's octagonal.

**Comment by 0-afflatus** [comment]
I think the texture should be less bright and contrasty, I agree with all that. 

The rest is really a matter of taste. I think the Pixel Box jungle trees are horrible personally. But my opinion isn't important here as I shall stick with VanessE's multicoloured textures. Committee-led design decisions tend to lead to camels and magnolia wallpaper in my experience.

**Comment by PilzAdam** [comment]
@paramat I'm not requesting to tweak the texture. I think the current texture is good enough.

**Comment by paramat** [comment]
PilzAdam, okay. The current texture is not good, it has no consistency with the side texture and the confirmed structure of the node.

Updated, see new images in first comment. Core is a little brighter and is a more orangey dark brown to appear closer to Rosewood and Wenge. The strangler fig lianas now have a brighter core to appear as cut cross-sections.

**Comment by paramat** [comment]
Because this is a texture i would like to have majority approval, so i am hoping for 2 approvals in addition to mine.

**Comment by Napiophelios** [comment]
the tree is to be round, the inner core rings might be round too.

**Comment by ObaniGemini** [comment]
I think the current texture is not good enough but this one is even worse to me. And I don't care about any lianas argument.

**Comment by paramat** [comment]
Napiophelios, it's not round it's octagonal, but more intended to be a square with chopped corners, i deliberately kept the rings square to keep a 'square' character. I usually oppose roundness in Minetest.

**Comment by paramat** [comment]
Approval from sfan5 http://irc.minetest.ru/minetest-dev/2016-03-04#i_4549673

**Comment by Gael-de-Sailly** [comment]
Agree with @ObaniGemini and @PilzAdam. This seems too contrasted. For me the previous texture was good.
However, for the *leaves* texture, the one @kilbith showed could be better.

**Comment by paramat** [comment]
f80fafbcfbca43bbb271b521fad5370cb064d788

**Comment by PilzAdam** [comment]
@paramat so we ignore my :-1: now?

Can you at least make the circle less regular? And maybe modify the jungle tree placement so that the bottom nodes are pointing sideways and not up?
